cabal.project

```diff
@@ -1,3 +1,15 @@
 packages: streamly-examples.cabal
 
 jobs: 1
+
+-- TODO: Comment the following after the streamly release
+
+source-repository-package
+    type:       git
+    location:   https://github.com/composewell/streamly
+    tag:        f1a04362162d707054115a34563e1c35199bb796
+
+source-repository-package
+    type:       git
+    location:   http://github.com/composewell/unicode-data
+    tag:        91bc99182b39290b423c4a584a0b077d91eead4c
```

Comment:
unicode-data is already released why do we need a source-repository-package for this?

Reply:
It's for local development. Ah, I should just use `cabal.project.local`